units/src/fee_rate/mod.rs

```diff
@@ -197,7 +198,10 @@ impl<'a> Arbitrary<'a> for FeeRate {
 
 #[cfg(test)]
 mod tests {
+    use alloc::format;
```

Comment:
There's a feature flag for this you can use: `#[cfg(feature = "alloc")]` for this.